config/jobs/README.md

```diff
@@ -1,9 +1,146 @@
 # Kubernetes Prow Job Configs
 
-Here are prow job configs for the Kubernetes prow deployment.
+All prow job configs for [prow.k8s.io] live here.
 
-Please add jobs per org|repo/sig subdir.
+They are tested and validated by tests in [`config/tests`](/config/tests)
 
-The file basename will be used as a configmap key so they need to be unique across this directory.
+## Job Cookbook
 
-Questions? Contact @krzyzacy!
\ No newline at end of file
+This document attempts to be a step-by-step easy copy-pasta guide to the use
```

Comment:
nit: there's a lot of compound adjectives stacked up here. Commas might help
Suggest: "a step-by-step, easy, and copy-pastable guide to using prow jobs..."

Reply:
done